README.md

```diff
@@ -5,7 +5,12 @@
 
 Want to see it in action? Check out the demo: https://simplewebrtc.com/demo.html
 
-Want to run it locally? `npm install && npm run test-page`
+Want to run it locally? 
+1 - Install stupid-server globally
```

Comment:
Oops, this one is my fault! Stupid server should not be required to be installed globally. I think instead we should list this as a devDependency, so it gets installed automatically. Would you mind changing this PR to do that? Thanks for your help! Then we could just change the readme back, and maybe add a `open your browser to https://0.0.0.0:8443/test/` line.